.circleci/config.yml

```diff
@@ -141,6 +142,28 @@ jobs:
           name: Run Docker Tests
           command: |
             docker-compose up --exit-code-from app
+  test:frontend:
+    working_directory: ~/project
+    docker:
+      - image: circleci/python:3.6-node
+        environment:
+          CONTAINER_MODE: TEST
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: ~/project
+      - restore_cache:
+          keys:
+            # when lock file changes, use increasingly general patterns to restore cache
+            - node-v1-{{ .Branch }}-{{ checksum "~/project/app/package-lock.json" }}
+            - node-v1-{{ .Branch }}-
+            - node-v1-
+      - run:
+          name: Run Frontend Tests
+          command: |
+            cd app
+            npm run test
```

Comment:
Split out front end tests from back end so CI runs faster
https://circleci.com/docs/2.0/parallelism-faster-jobs/

Reply:
parallelizing looks good..also i think the status check for "ci/circleci: test" needs to be removed

Reply:
good point - will leave active until this PR is ready to merge in - so that the other branches keep working!

Reply:
sounds good! will give it a full look this weekend